include/fmt/chrono.h

```diff
@@ -451,6 +451,27 @@ FMT_END_DETAIL_NAMESPACE
 template <typename Char, typename Duration>
 struct formatter<std::chrono::time_point<std::chrono::system_clock, Duration>,
                  Char> : formatter<std::tm, Char> {
+  FMT_CONSTEXPR FMT_INLINE void generate_default_spec(char) {
```

Comment:
`FMT_INLINE` is not needed here, please remove.

Also it's better to move this function to the detail namespace since it doesn't depend on template parameters and is not part of the public API.

Reply:
But it's a member function. Moving it to the detailed namespace will make it be templated. It looks like

```cpp
template <typename Char, typename Duration>
FMT_CONSTEXPR void generate_default_spec(
    formatter<std::chrono::time_point<std::chrono::system_clock, Duration>,
              Char>* formatter,
    char) {
  formatter->specs = {"%Y-%m-%d %H:%M:%S", 17};
}
```

Maybe making it a private function is more tidier?

```cpp
private:

FMT_CONSTEXPR void generate_default_spec(char) {    
    this->specs = {"%Y-%m-%d %H:%M:%S", 17};
}
```